bindings/python/synthizer.pyi

```diff
@@ -0,0 +1,260 @@
+from enum import Enum
+from typing import Iterator, List, Optional, Tuple, Union
+
+StringOrBytes = Union[str, bytes]
+Double3 = Tuple[float, float, float]
+Double6 = Tuple[float, float, float, float, float, float]
+EventSourceType = Union['Source', 'Generator']
+
+
+class LoggingBackend(Enum):
+    STDERR: int
+
+
+STDERR: LoggingBackend = LoggingBackend.STDERR
+
+
+class LogLevel(Enum):
+    DEBUG: int
+    ERROR: int
+    INFO: int
+    WARN: int
+
+
+DEBUG: LogLevel = LogLevel.DEBUG
+INFO: LogLevel = LogLevel.INFO
+ERROR: LogLevel = LogLevel.ERROR
+WARN: LogLevel = LogLevel.WARN
+
+
+class DistanceModel(Enum):
+    NONE: int
+    LINEAR: int
+    EXPONENTIAL: int
+    INVERSE: int
+
+
+NONE: DistanceModel = DistanceModel.NONE
+LINEAR: DistanceModel = DistanceModel.LINEAR
+EXPONENTIAL: DistanceModel
+INVERSE: DistanceModel
+
+
+class NoiseType(Enum):
+    FILTERED_BROWN: int
+    UNIFORM: int
+    VM: int
+
+
+FILTERED_BROWN: NoiseType = NoiseType.FILTERED_BROWN
+UNIFORM: NoiseType = NoiseType.UNIFORM
+VM: NoiseType = NoiseType.VM
+
+
+class PannerStrategy(Enum):
+    HRTF: int
+    STEREO: int
+
+
+HRTF: PannerStrategy = PannerStrategy.HRTF
+STEREO: PannerStrategy = PannerStrategy.STEREO
+
+# Initialization functions:
+
+
+def initialized() -> Iterator[None]: ...
+
+
+def initialize() -> None: ...
+
+
+def shutdown() -> None: ...
+
+
+# Functions for configuring logging:
+
+
+def configure_logging_backend(backend: LoggingBackend) -> None: ...
+
+
+def set_log_level(level: LogLevel) -> None: ...
+
+
+class _BaseObject:
```

Comment:
This isn't part of the public API, and shouldn't be used by users. Is it necessary to type it?